Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/viewholders/GameViewHolder.java

```diff
@@ -24,15 +20,15 @@ public class GameViewHolder extends RecyclerView.ViewHolder
 
   public GameFile gameFile;
 
-  public GameViewHolder(View itemView)
+  public GameViewHolder(@NonNull CardGameBinding binding)
   {
-    super(itemView);
+    super(binding.getRoot());
 
-    itemView.setTag(this);
+    binding.getRoot().setTag(this);
 
-    imageScreenshot = itemView.findViewById(R.id.image_game_screen);
-    textGameTitle = itemView.findViewById(R.id.text_game_title);
-    textGameTitleInner = itemView.findViewById(R.id.text_game_title_inner);
-    textGameCaption = itemView.findViewById(R.id.text_game_caption);
+    imageScreenshot = binding.imageGameScreen;
+    textGameTitle = binding.textGameTitle;
+    textGameTitleInner = binding.textGameTitleInner;
+    textGameCaption = binding.textGameCaption;
```

Comment:
Could be removed.

Though, hmm... Does this class have any purpose left after removing these variables?